lib/cn/intern_builder.c

```diff
@@ -171,13 +184,20 @@ ib_lcp_len(struct intern_level *ib, const struct key_obj *ko)
     struct intern_key *k;
     uint               new_pfxlen, old_pfxlen;
 
+#if HSE_KVS_KEY_CPE_ALIGNMENT == 8
+#define memlcp   memlcpq
+#endif
```

Comment:
Hate to this appear in multiple source files.

Reply:
Agreed.  But I wanted it close by so it's obvious.  That said, I could move it to arch.h and give these memlcp() calls a different name, like memlcp_cpe() or something, then it's in one place under control of the define.

Reply:
Hmm, arch.h is wrong, cn_internal.h maybe...

Reply:
Moved to wbt_builder.h